src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -517,7 +517,7 @@ export class MatAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
   }
 
   private _getConnectedElement(): ElementRef {
-    return this._formField ? this._formField._connectionContainerRef : this._element;
+    return this._formField ? this._formField.getPopupConnectionElementRef() : this._element;
```

Comment:
How about `getConnectedOverlayOrigin`? I want to save the term "popup" in case we ever make one as its own thing (a la xap-bubble). We also use the term "origin" throughout the overlay positioning code to refer to this element.